landing-page/src/components/landing-page-logs.ts

```diff
@@ -255,9 +265,14 @@ class LandingPageLogs extends LitElement {
     }
   }
 
-  private async _downloadFullLog() {
+  private _downloadLogs() {
     const timeString = new Date().toISOString().replace(/:/g, "-");
-    fileDownload("/observer/logs", `home-assistant_${timeString}.log`);
+    const downloadUrl = this._observerFallback
```

Comment:
Is there a benefit of not using the observer for downloading the logs?

Reply:
I'll recheck

Reply:
Checked, yes download observer logs is even better here, because it downloads all logs.